app.css

```diff
@@ -0,0 +1,21 @@
+* {
+  box-sizing: border-box;
+
+}
+.navbar {
+  display: inline-block;
+  width: 100%;
+  overflow: auto;
+  background: #64b5f6;
+  color: white;
+
+}
+.nav_bar-left  {
+
+  text-align: left;
+}
+
+.nav_bar-right  {
+
+  text-align: right;
```

Comment:
Look at using the `float` property to line up and pin the left and right of the nav elements. See the grouping suggestion in the HTML